easybuild/easyconfigs/s/SPH-EXA/SPH-EXA-v0.2.2-gompi-2021a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = "CMakeMake"
+
+name = "SPH-EXA"
+version = "v0.2.2"
```

Comment:
@jgphpc Do you mind stripping out the `v` here, to have a clean version for the installation?

Reply:
https://github.com/easybuilders/easybuild-easyconfigs/pull/13244/commits/16d74b36396d706e5d35d6a0053b74df46c35876